### PR TITLE
Add 64 bits optionnal check for DXR  wizard (for DXR player)

### PR DESCRIPTION
**Pull request by remi-chapelain**
### Purpose of this PR
RTX effects needs to have a 64 bits architecture for them to work in player but it was not mentioned anywhere. 
[This has just been added to the doc](https://github.com/Unity-Technologies/Graphics/pull/1631) but since users don't read the doc, adding it to the optional check in the DXR wizard would be a great addition to what's already there. 

image

image

---
### Testing status
Not a lot to test there. 
I just tried to change architecture to see if that is affecting correctly the check

### Comments to reviewers
Just want to make sure that it makes sense and I did not make obvious mistakes.

### CODE REVIEW

**Comment by iM0ve** [comment]
Good addition, adds an informative text as in the images below for DXR.

Tested:
* The warning is visible when 32 bit is selected
* Warning is not visible in other non DXR menus
* Warning turns to green checkbox when build is set to 64 

Suggestion:
I think this option would benefit from having Fix button. Is there a downside why user would want to keep 32 bits in his DXR project? If 32 vs 64 is not an option, because 32 just doesnt work we should include this fix in the "Fix all" list if possible.

**Comment by remi-chapelain** [comment]
Following @iM0ve comment, I have changed the check from optional to required and added a fix button for it. 
I've check the fix button on a fresh new project and also if it works with the top "fix all" button.

image

**Review by iM0ve — CHANGES_REQUESTED** [review]
Can we address this warning?

image

**Comment by remi-chapelain** [comment]
Obsolete function request fixed !

**Review by iM0ve — APPROVED** [review]
All seems good now

**Comment by RSlysz** [comment]
> 
> 
> Following @iM0ve comment, I have changed the check from optional to required and added a fix button for it.
> I've check the fix button on a fresh new project and also if it works with the top "fix all" button.
> 
> image

You just broke the resources group :O Please put the "Architecture 64 bits" outside of "DXR resources" (before or after but not inside especially if not indented.

File: com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs [hunk 1]
```diff
@@ -174,6 +174,8 @@ Entry[] entries
                         new Entry(InclusiveScope.DXR, Style.dxrStaticBatching, IsDXRStaticBatchingCorrect, FixDXRStaticBatching),
                         new Entry(InclusiveScope.DXR, Style.dxrActivated, IsDXRActivationCorrect, FixDXRActivation),
                         new Entry(InclusiveScope.DXR, Style.dxrResources, IsDXRAssetCorrect, FixDXRAsset),
+                        new Entry(InclusiveScope.DXR, Style.dxr64bits, IsArchitecture64Bits, FixArchitecture64Bits),
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs:177]: Move this line one line above to not break grouping of "DXR Resources"

**Comment by remi-chapelain** [comment]
> You just broke the resources group :O Please put the "Architecture 64 bits" outside of "DXR resources" (before or after but not inside especially if not indented.

Did not realized thoses were grouped ! I moved the 64 bits thingy above ! Thanks for pointing it out.

**Review by anisunity — APPROVED** [review]
lgtm

**Review by RSlysz — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**